README.md

```diff
@@ -59,41 +52,16 @@ pip to install directly from the master branch on github:
 ```
 pip install git+https://github.com/keras-team/keras-nlp.git --upgrade
 ```
-## Using KerasNLP with Keras Core
-
-As of version `0.6.0`, KerasNLP supports multiple backends with Keras Core out 
-of the box. There are two ways to configure KerasNLP to run with multi-backend 
-support:
-
-1. Via the `KERAS_BACKEND` environment variable. If set, then KerasNLP will be 
-using Keras Core with the backend specified (e.g., `KERAS_BACKEND=jax`).
-2. Via the `.keras/keras.json` and `.keras/keras_nlp.json` config files (which 
-are automatically created the first time you import KerasNLP):
-   - Set your backend of choice in `.keras/keras.json`; e.g., `"backend": "jax"`. 
```

Comment:
I've dropped any mention of `.keras` json files for now. There is too much to focus on!

Let's only document `KERAS_BACKEND` environment variable for now, and try to make that less confusing. After Keras 3 is out, we can go back to documenting persistent settings.